src/collective/vdexvocabulary/treevocabulary.rst

```diff
@@ -54,13 +54,13 @@ check translations::
 
     >>> from zope.i18n import translate
     >>> translate(vdex_vocab.term_by_value['nwe'].title)
-    u'North-west of Europe'
+    'North-west of Europe'
```

Comment:
This should return unicode on Py2.7... I think something like https://github.com/plone/plone.indexer/blob/50a0ead2afba37241e21db1ed04985f06ec0f22c/plone/indexer/tests.py#L25 might be needed